src/core/biblio.js

```diff
@@ -107,14 +107,15 @@ export async function run(conf) {
   conf.biblio = biblio;
   const localAliases = Array.from(Object.keys(conf.localBiblio))
     .filter(key => conf.localBiblio[key].hasOwnProperty("aliasOf"))
-    .map(key => conf.localBiblio[key].aliasOf);
+    .map(key => conf.localBiblio[key].aliasOf)
+    .filter(key => !conf.localBiblio.hasOwnProperty(key));
```

Comment:
I'm unsure about this change... can you help me understand it a bit?

Reply:
These localAliases are added later on to the list of references to fetch from the network / db, since they could refer to external references. The filter I added ensures we only fetch those, and not the local aliases of local references. For tests where the entire bibliography is local, this avoids lookups.

Alternatively, we could also keep `localAliases` as before but swap the order of the `concat` and `filter` in the computation of `neededRefs`.

Reply:
Thanks for the clarification. I’m ok to go with this.